stl/inc/bitset

```diff
@@ -336,29 +337,12 @@ public:
     }
 
     _NODISCARD size_t count() const noexcept { // count number of set bits
-        const char* const _Bitsperbyte = "\0\1\1\2\1\2\2\3\1\2\2\3\2\3\3\4"
-                                         "\1\2\2\3\2\3\3\4\2\3\3\4\3\4\4\5"
-                                         "\1\2\2\3\2\3\3\4\2\3\3\4\3\4\4\5"
-                                         "\2\3\3\4\3\4\4\5\3\4\4\5\4\5\5\6"
-                                         "\1\2\2\3\2\3\3\4\2\3\3\4\3\4\4\5"
-                                         "\2\3\3\4\3\4\4\5\3\4\4\5\4\5\5\6"
-                                         "\2\3\3\4\3\4\4\5\3\4\4\5\4\5\5\6"
-                                         "\3\4\4\5\4\5\5\6\4\5\5\6\5\6\6\7"
-                                         "\1\2\2\3\2\3\3\4\2\3\3\4\3\4\4\5"
-                                         "\2\3\3\4\3\4\4\5\3\4\4\5\4\5\5\6"
-                                         "\2\3\3\4\3\4\4\5\3\4\4\5\4\5\5\6"
-                                         "\3\4\4\5\4\5\5\6\4\5\5\6\5\6\6\7"
-                                         "\2\3\3\4\3\4\4\5\3\4\4\5\4\5\5\6"
-                                         "\3\4\4\5\4\5\5\6\4\5\5\6\5\6\6\7"
-                                         "\3\4\4\5\4\5\5\6\4\5\5\6\5\6\6\7"
-                                         "\4\5\5\6\5\6\6\7\5\6\6\7\6\7\7\x8";
-        const unsigned char* _Ptr       = &reinterpret_cast<const unsigned char&>(_Array);
-        const unsigned char* const _End = _Ptr + sizeof(_Array);
-        size_t _Val                     = 0;
-        for (; _Ptr != _End; ++_Ptr) {
-            _Val += _Bitsperbyte[*_Ptr];
+        // local copy allows the compiler to move the condition out of loop

```

Comment:
This could be `constexpr`

Reply:
No. __isa_available is runtime detected.

Reply:
and the whole `count()` is not `constexpr` in standard